app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -17,7 +17,7 @@ def textual_group_relationships
   end
 
   def textual_group_status
-    TextualGroup.new(_("Status"), %i(refresh_status refresh_date))
+    TextualGroup.new(_("Status"), %i(refresh_status refresh_date authentication_status))
```

Comment:
afaik the order of those fields in other providers is that auth status is displayed first - before the refresh fields.

Reply:
Yes, you are right. Changing it.